### PR TITLE
refs #107 provide clean method with AutocompleteUrlProxy

### PR DESCRIPTION
**Pull request by moumoutte**

### CODE REVIEW

File: demo/views_proxy.py [hunk 1]
```diff
@@ -180,3 +180,22 @@ def item(request, pk):
...
+
+@require_GET
+def atomic_item(request, pk):
+    """
+    Similar to `item` but does not return a list
+    """
+    data = None
+    for item in DATABASE:
+        if text(item['pk']) == text(pk):
+                data = convert_data(item)
```
Comment (brunobord) [inline demo/views_proxy.py]: isn't this one a bit too indented?

File: agnocomplete/exceptions.py [hunk 2]
```diff
@@ -37,3 +37,7 @@ class SkipItem(Exception):
     Occurs when Item has to be skipped when building the final Payload
     """
     pass
+
+
+class ItemNotFound(Exception):
+    pass
```
Comment (brunobord) [inline agnocomplete/exceptions.py:46]: missing docstring here

**Review by brunobord — CHANGES_REQUESTED** [review]
Missing changelog item and docstring. But overall, it looks okay.

**Comment by moumoutte** [comment]
Changes have been done as requested.

**Review by brunobord — APPROVED** [review]

**Comment by brunobord** [comment]
Shall we merge using the ``--fast-forward` option?

**Comment by moumoutte** [comment]
I guess !

**Pull request merged**